ONSdigital/dp-release-calendar-api
# Implement legacy endpoint

### What

Implement `legacy` endpoint returning releases coming from Zebedee

https://trello.com/c/2Y4LxMHr/743-create-legacy-handler-for-dpreleasecalendarapi

### How to review

Check code and tests are correct.
Optionally test it locally:
- Run latest versions of [zebedee](https://github.com/ONSdigital/zebedee) and [dp-api-router](https://github.com/ONSdigital/dp-api-router)
- `make debug`
- Send a GET call to `http://localhost:27800/releases/legacy?url={release_url}` where `release_url` is a valid release path in your zebedee instance (for example: `/releases/adoption`)
- Inspect the returned json

### Who can review

Anyone not me
